x/shield/types/types.go

```diff
@@ -6,85 +6,53 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// Pool contains a shield pool's data.
+// Variables for the shield global pool
+
+// TotalCollateral is the amount of total collaterals in the shield module.
+type TotalCollateral sdk.Int
```

Comment:
Oh I put these types, but I think we should remove them. If we use these, we have to cast the variables every time we do arithmetic.

Reply:
I think I'll just keep it for now. We can make some structure like `GlobalPool` to include them all if we decide to do so.